android/app/src/main/java/com/rohtvapp/ROHBitMovinPlayerPackage/PlayerContainerView.java

```diff
@@ -90,13 +91,15 @@ public void init() {
         audioCodecPriority.add("mp4a.a5");
         audioCodecPriority.add("mp4a.40");
 
+        boolean shouldEnableTunneledPlayback = Build.MANUFACTURER.contains("Google") || Build.MANUFACTURER.contains("Amazon");
```

Comment:
I'm in favour of refactoring this into something like `isTargettedDevice()`, which we reuse for default quality setting as well as tunnelled playback.

Reply:
Sure - will do now.